### PR TITLE
Update platform project and board layer for IMXRT1050-EVKB

### PR DESCRIPTION
**Pull request by ZigaMahne**

### CODE REVIEW

**Comment by grasci-arm** [comment]
Can one of the admins verify this patch?

**Comment by grasci-arm** [comment]
Can one of the admins verify this patch?

File: layer/Board/IMXRT1050-EVKB/Board.clayer [hunk 1]
```diff
@@ -82,7 +82,7 @@
     <component Cclass="CMSIS" Cgroup="CORE" Cvendor="ARM" layer="Board"/>
     <component Cclass="CMSIS Driver" Cgroup="Ethernet MAC" Cvendor="Keil" layer="Board"/>
     <component Cclass="CMSIS Driver" Cgroup="Ethernet PHY" Csub="KSZ8081RNA" Cvendor="Keil" layer="Board">
-      <cflags add="-DETH_PHY_REF_CLK_50M=&quot;1&quot;" compiler="AC6"/>
+      <cflags add="-DETH_PHY_REF_CLK_50M=&quot;1&quot;" compiler="AC6" remove="-g"/>
```
Comment (VladimirUmek) [inline layer/Board/IMXRT1050-EVKB/Board.clayer:85]: Why remove="-g"?

File: layer/Board/IMXRT1050-EVKB/Board.clayer [hunk 2]
```diff
@@ -119,7 +119,7 @@
     </component>
     <component Cclass="Device" Cgroup="SDK Utilities" Csub="assert" Cvendor="NXP" layer="Board"/>
     <component Cclass="Device" Cgroup="SDK Utilities" Csub="debug_console" Cvendor="NXP" layer="Board">
-      <cflags add="-DSDK_DEBUGCONSOLE=&quot;1&quot; -D__MCUXPRESSO" compiler="AC6"/>
+      <cflags add="-DSDK_DEBUGCONSOLE=&quot;1&quot; -D__MCUXPRESSO" compiler="AC6" remove="-g"/>
```
Comment (VladimirUmek) [inline layer/Board/IMXRT1050-EVKB/Board.clayer:122]: Also here, why remove="-g"?

File: layer/Board/IMXRT1050-EVKB/Board.clayer (hunk 1 above)
Comment (ZigaMahne) [reply layer/Board/IMXRT1050-EVKB/Board.clayer]: It look like .. remove="-g" .. is bug from MDK 5.38, so I reverted this commit.

**Pull request merged**